build.gradle

```diff
@@ -13,15 +13,22 @@ configurations {
     provided
 }
 
-version = "0.1.17"
+version = {
+     def vd = versionDetails()
+     if (vd.commitDistance == 0 && vd.lastTag ==~ /^[0-9]+\.[0-9]+\.[0-9]+(\.[a-zA-Z0-9]+)?/) {
+         vd.lastTag
+     } else {
+         "0.0.0.${vd.gitHash}.pre"
+     }
+ }()
 
 sourceCompatibility = 1.8
 targetCompatibility = 1.8
 
 dependencies {
     compile  "org.embulk:embulk-core:0.9.17"
     provided "org.embulk:embulk-core:0.9.17"
-    compile group: "com.facebook.business.sdk", name: "facebook-java-business-sdk", version: "16.0.2"
+    compile group: "com.facebook.business.sdk", name: "facebook-java-business-sdk", version: "15.0.0"
```

Comment:
Is this change necessary?

Reply:
Thanks for your comment.
The change is unnecessary...
I fixed it.  [10b47a2](https://github.com/trocco-io/embulk-input-facebook_ads_insights/pull/19/commits/10b47a2ee5c101938daa2e3fec9ef35a86d73686)